src/kyrios.py

```diff
@@ -32,10 +34,26 @@ def readPackage(filename, context):
 
     context['packages'][key] = packageMetadata
 
+def getInstallPlatform(context, package):
+    simplifiedPlatform = context['simplifiedPlatform']
```

Comment:
Make this just `installPlatform = context['simplifiedPlatform`] and fix the fatal log line to also use installPlatform.

Reply:
Ah, good point. I'm not sure why I had it that way other than I just moved the code from its original location.

Reply:
Actually, I want the fatal log line to specify simplifiedPlatform rather than installPlatform - by the point that the message is issued, installPlatform would always be `'generic'` otherwise, which clearly isn't right.